api.json

```diff
@@ -0,0 +1,732 @@
+{
+    "openapi": "3.0.1",
+    "info": {
+        "title": "Deku",
+        "version": "v1",
+        "description": "API for deku v1."
+    },
+    "paths": {
+        "/chain/info": {
+            "get": {
+                "description": "Returns some informations about the chain: the consensus and discovery contract address.",
+                "tags": [
+                    "chain"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "consensus and discovery address",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/ChainInfo"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/chain/level": {
+            "get": {
+                "description": "Returns the current level/height of the chain.",
+                "tags": [
+                    "chain"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returns the current level of the chain as a string representing an int64.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "level": {
+                                            "$ref": "#/components/schemas/BlockLevel"
+                                        }
+                                    }
+                                },
+                                "examples": {
+                                    "level": {
+                                        "$ref": "#/components/examples/level-response"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/chain/blocks": {
+            "get": {
+                "description": "Streams blocks starting at the current level of the chain",
+                "tags": [
+                    "chain"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "blocks",
+                        "headers": {
+                            "Transfer-Encoding": {
+                                "schema": {
+                                    "type": "string",
+                                    "description": "Used only to stream blocks.",
+                                    "example": "chunked"
```

Comment:
This endpoint aims to stream blocks,
Users will be able to connect to this http stream and receive every block applied by the chain. So that users will have the ability to listen/subscribe to the applied blocks.
When a user open an http stream, the first streamed block is the block at the current level of the chain, (the next block is then at level + 1, etc...)
But I still need to know if an http stream will impact the performance of the chain.